src/web/assets/cp/src/js/CP.js

```diff
@@ -524,7 +522,10 @@ Craft.CP = Garnish.Base.extend(
       );
     },
 
-    toggleSidebar: function () {
+    toggleSidebar: function ({target}) {
```

Comment:
If we need to accept a target here, it should just be a regular `target` argument, rather than an object. Then update the event handler to extract the target from the event data.

```js
this.addListener($('#sidebar-toggle'), 'click', ({target}) => {
  this.toggleSidebar(target);
});
```

Also, the function should continue to work if `target` isn’t passed in, for backwards compatibility.

Reply:
For the `target` variable, I thought I'd use a destructuring assignment to pull it out of the event object, to cut down on code inside the `toggleSidebar` function. I can re-work that though 👍🏼 

Out of curiosity, what browsers/versions don't support `target` in this context? I'm not really versed on issues that would arise there. Appreciate the feedback!

Reply:
Not a browser support issue; just an API clarity issue. `toggleSidebar()` should be something that can be called directly, without needing to pass an event object into.

Reply:
Makes sense! I'll keep that in mind going forward